src/components/Projects/Section/sections/Embed/styled.ts

```diff
@@ -0,0 +1,16 @@
+import styled from '@theme'
+
+export const Wrapper = styled.div`
+  position: relative;
+  padding-bottom: 56.25%; /* 16:9 */
+  padding-top: 25px;
```

Comment:
why this padding-top? I believe we don't have padding in any section and the sectionWrapper takes care of all the top and bottom spacings

Reply:
These styles are about this: https://css-tricks.com/NetMag/FluidWidthVideo/Article-FluidWidthVideo.php